README.md

```diff
@@ -48,15 +48,7 @@ Samson will:
 
 ### Contributing
 
-Improvements are always welcome. Please follow these steps to contribute
-
-1. Submit a Pull Request with a detailed explanation of changes and screenshots (if UI is changing)
-1. Receive a :+1: from a core team member
-1. Core team will merge your changes
-
-### Team
-
-Core team is [@henders](https://github.com/henders), [@grosser](https://github.com/grosser), [@jonmoter](https://github.com/jonmoter), [@irwaters](https://github.com/irwaters).
+Issues and Pull Requests are always welcome, submit your idea!
```

Comment:
Sounds too simplify..shouldn't Samson follow the basic rules? https://zendesk.atlassian.net/wiki/spaces/ENG/pages/264288863/PROJECT+Templates#PROJECTTemplates-CONTRIBUTING

Reply:
I don't want to scare contributors / overwhelm them in the readme (very few people will contribute, so little value in pasting big blobs of test here) ... we have a PR template for more details

Reply:
I think there's also a special file you can create like CONTRIBUTING.md or something that Github will automatically link to in the PR view for new users...

https://help.github.com/en/articles/setting-guidelines-for-repository-contributors

Reply:
the PR template has the steps users should take and it's more "in your face" than a link to another doc, so picking that for now until we need to explain more